src/backend/core/api/viewsets.py

```diff
@@ -1062,13 +1062,12 @@ class ItemAccessViewSet(
     def perform_create(self, serializer):
         """Add a new access to the item and send an email to the new added user."""
         access = serializer.save()
-        language = self.request.headers.get("Content-Language", "en-us")
 
         access.item.send_invitation_email(
             access.user.email,
             access.role,
             self.request.user,
-            language,
+            self.request.user.language or settings.LANGUAGE_CODE,
```

Comment:
```suggestion
            self.request.user.language,
```

It is not enough ? The default language is already `settings.LANGUAGE_CODE` and with the permission you must be connected.
But maybe I'm missing something